avaje-config/src/main/java/io/avaje/config/ConfigServiceLoader.java

```diff
@@ -21,40 +27,77 @@ static ConfigServiceLoader get() {
   private final ModificationEventRunner eventRunner;
   private final List<ConfigurationSource> sources = new ArrayList<>();
   private final List<ConfigurationPlugin> plugins = new ArrayList<>();
-  private final Parsers parsers;
+  private final Map<String, URIConfigLoader> uriLoaders;
+  private final Map<String, ConfigParser> parsers;
 
   ConfigServiceLoader() {
-    ModificationEventRunner _eventRunner = null;
-    ConfigurationLog _log = null;
-    ResourceLoader _resourceLoader = null;
+    ModificationEventRunner spiEventRunner = null;
+    ConfigurationLog spiLog = null;
+    ResourceLoader spiResourceLoader = null;
     List<ConfigParser> otherParsers = new ArrayList<>();
+    List<URIConfigLoader> loaders = new ArrayList<>();
 
     for (var spi : ServiceLoader.load(ConfigExtension.class)) {
       if (spi instanceof ConfigurationSource) {
         sources.add((ConfigurationSource) spi);
       } else if (spi instanceof ConfigurationPlugin) {
         plugins.add((ConfigurationPlugin) spi);
-      } else if (spi instanceof ConfigParser) {
+      } else if (spi instanceof ConfigParser && !Boolean.getBoolean("skipCustomParsing")) {
```

Comment:
`skipCustomParsing`

What is this for? Unit test?

Reply:
It's possible that someone won't want to have custom parsing